web/src/compose_validate.ts

```diff
@@ -54,6 +54,20 @@ function set_message_too_long(status: boolean): void {
     update_send_button_status();
 }
 
+function set_message_too_long_for_edit(status: boolean, $container: JQuery): void {
```

Comment:
I think it would help clarify why `set_message_too_long` is different if you added a prep commit to rename it to `set_message_too_long_for_compose`

Reply:
Added a separate PR for the same.